pkg/handler/v1/provider/openstack.go

```diff
@@ -315,7 +278,33 @@ type OpenstackTenantReq struct {
 	// Domain OpenStack domain name
 	Domain string
 	// in: header
-	// DatacenterName Openstack datacenter na
+	// DatacenterName Openstack datacenter name
+	DatacenterName string
+	// in: header
+	// Credential predefined Kubermatic credential name from the presets
+	Credential string
+}
+
+// OpenstackReq represent a request for openstack
+// swagger:parameters listOpenstackSizes listOpenstackNetworks listOpenstackSecurityGroups listOpenstackAvailabilityZones
+type OpenstackReq struct {
+	// in: header
+	// Username OpenStack user name
+	Username string
+	// in: header
+	// Password OpenStack user password
+	Password string
+	// in: header
+	// Domain OpenStack domain name
+	Domain string
+	// in: header
+	// Tenant OpenStack tenant name
+	Tenant string
+	// in: header
+	// TenantID OpenStack tenant ID
+	TenantID string
+	// in: header
+	// DatacenterName Openstack datacenter name
```

Comment:
This change does not seem to be required. Isn't it?

Reply:
I just placed OpenstackReq close to OpenstackTenantReq for readability. Should I undo it?

Reply:
no, you can keep it